apps/src/util/useFetch.js

```diff
@@ -93,6 +96,7 @@ export const useFetch = (url, options) => {
   return {
     loading: fetchState.loading,
     data: fetchState.data,
+    response: fetchState.response,
```

Comment:
I ended up not needing the response, but left this change here because I thought it could be useful for other callers of `useFetch` in the future?  Also ok to revert the `response` changes (but keep the `EMPTY_OPTIONS` changes).